test/testdata/rewriter/quoted_symbol_error.rb

```diff
@@ -0,0 +1,10 @@
+# typed: true
+
+class A < T::Struct
+  prop :'foo', Integer
+  prop :"bar", Integer
```

Comment:
```suggestion
  prop :"bar", Integer
  prop :"", Integer
  prop :"#{A.name}", Integer
```

Reply:
can you update the test to include these edge cases? I think that the Prop dsl currently ignores those so _probably_ the code passes as is with these suggestions but I'm not sure.

Reply:
Not sure how to call the constructor on `A` with `:""` and `:"#{A.name}"`, currently tried the following and getting an error:
```
test/testdata/parser/quoted_symbol_error.rb:10: ERROR: Found unexpected error:
10: a = A.new(foo: 0, bar: 1, :"" => 2, :"#{A.name}" => 3)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ error: Cannot call `A#initialize` with a `Hash` keyword splat because the method has required keyword parameters
```

Reply:
You can add a test to capture the error without doing anything extra and only passing foo and bar.

Reply:
(Basically just leaving that part of the test unchanged)

Reply:
Test files don’t have to be free of errors—capturing error behavior is the point of tests.

Reply:
ack done